client/stylesheets/components/_footer-help.scss

```diff
@@ -1,4 +1,4 @@
-.footer-help__round {
+.footer-help__alwayson {
```

Comment:
Also, following ```BEM```, ```on``` is a ```modifier```, so name should rather be :

```css
.footer__help--on
```

or

```css
footer-help--on
```

Reply:
For the color it's supposed to be less visible than a CTA, so it's not white
For the animation, it might be not perfect but it's good enough, let's iterate later.